docs/when.rst

```diff
@@ -30,61 +21,36 @@ Merging the two task lists is left to the user.
 A better idea would've been to use ``git`` to synchronize the ``todo.txt``
 file, which is at least able to resolve some basic conflicts.
```

Comment:
todo.txt actually has multiple Android apps and at least one iOS app, so I don't think DAV has any advantage there.

On 2 February 2016 00:10:42 CET, Hugo Osvaldo Barrera <notifications@github.com> wrote:
>> @@ -30,61 +21,36 @@ Merging the two task lists is left to the user.
>>  A better idea would've been to use ``git`` to synchronize the
>``todo.txt``
>>  file, which is at least able to resolve some basic conflicts.
>
>Also worth mentioning the lack of clients for lots of
>devices/platforms, while *dav support is kinda universal.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/untitaker/vdirsyncer/pull/327/files#r51497092

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.